### ISSUE TITLE
Ecoregions data

### ISSUE THREAD

**Issue by lhilarides** [Vizzuality/mangrove-atlas#741]
Here is the latest version of the ecoregions geometries
[Mangrove_EcoregionsL4_v2023.zip](https://github.com/Vizzuality/mangrove-atlas/files/11488358/Mangrove_EcoregionsL4_v2023.zip)

**Comment by AngelArcones** [comment]
@lhilarides geometries look fine, but I would need clarification on the meaning (and relevance) of some of the variables: Shape_Leng, Shape_Le_1, Shape_area (are these coastal length and area?) and native (only 2 cases with "0", is this relevant?)

**Comment by klongleywood** [comment]
@AngelArcones here are the data that accompanies the ecoregion data. We suggest the widget display the data from the spreadsheet in a way that's similar to the image pasted below, with a placeholder to link to the pdf of the ecoregional assessment for each location. 

[Ecoregion RedList Data.xlsx](https://github.com/Vizzuality/mangrove-atlas/files/11851172/Ecoregion.RedList.Data.xlsx)

![image](https://github.com/Vizzuality/mangrove-atlas/assets/54109064/349a9c1f-8e70-4cfd-bbfb-837c4ac2a710)

**Comment by dhakelila** [comment]
Hi, @klongleywood , is that ok if we work on this next week? I want to make sure that the on-going pieces of the project are done by the end of the week.

**Comment by klongleywood** [comment]
@AngelArcones this may be a question for @lhilarides -- I thought we were only planning to show 5 ecoregions?

**Comment by lhilarides** [comment]
> @AngelArcones this may be a question for @lhilarides -- I thought we were only planning to show 5 ecoregions?

That is correct, for now we only show the 5 ecoregions on the map that have data associated with them

**Comment by klongleywood** [comment]
@AngelArcones for this widget we'd also like links to the associated pdf reports.  These are draft versions for now but assuming it's relatively easy to update the pdfs when they are available?


[RLE Mangroves of the Sunda Shelf.pdf](h
[RLE Mangroves of the Western Coral Triangle.pdf](https://github.com/Vizzuality/ma
[RLE Mangroves of the Andaman.pdf](https://github.com/Vizzuality/mangrove
[RLE Mangroves of the Bay of Bengal (1).pdf](https://github.com/Vizzuality/mangrove-atlas/files/11983087/RLE.Mangroves.of.the.Bay.of.Bengal.1.pdf)
-atlas/files/11983084/RLE.Mangroves.of.the.Andaman.pdf)
ngrove-atlas/files/11983083/RLE.Mangroves.of.the.Western.Coral.Triangle.pdf)
ttps://github.com/Vizzuality/mangrove-atlas/files/11
[RLE Mangroves of the South China Sea.pdf](https://github.com/Vizzuality/mangrove-atlas/files/11983085/RLE.Mangroves.of.the.South.China.Sea.pdf)
983082/RLE.Mangroves.of.the.Sunda.Shelf.pdf)
